### PR TITLE
blog link: How IBM watsonx Assistant uses Knative Eventing to train machine learning models

### PR DESCRIPTION
**Pull request by pierDipi**
Add Featured blog post with the link to the official CNCF case study https://www.cncf.io/case-studies/ibmwatsonxassistant/ and the visual created by @firatbezir and @zainabhusain227

### CODE REVIEW

File: blog/docs/articles/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# How IBM Watsonx Assistant uses Knative Eventing to train machine learning models
+
+![How IBM Watsonx Assistant uses Knative Eventing to train machine learning models](./images/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.png)
+
+Read the full case study on the CNCF website: https://www.cncf.io/case-studies/ibmwatsonxassistant/
+
+Thanks to [Firat Bezir](https://www.linkedin.com/in/firatbezir/)
+and [Zainab Husain](https://www.linkedin.com/in/zainabhusain1/) for creating the impact visual
+summary.
```
Comment (pierDipi) [inline blog/docs/articles/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.md:9]: @firatbezir and @zainabhusain227 can you confirm if these links are correct?

**Comment by aliok** [comment]
We also add this to the case studies page: https://github.com/knative/docs/pull/5987

But, I think it is ok to have it in there and also in blog

File: blog/docs/articles/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.md (hunk 1 above)
Comment (firatbezir) [reply blog/docs/articles/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.md:9]: it is correct :)

**Comment by pierDipi** [comment]
I didn't see that one :)

**Review by Cali0707 — APPROVED** [review]
/lgtm

**Pull request merged**